integration_tests/dbt_project.yml

```diff
@@ -59,6 +59,8 @@ seeds:
 
 vars:
   # ensure integration tests run successfully when there are 0 of a given model type (extra)
+  exclude_packages: []
```

Comment:
Do we feel okay not having a package to exclude for our integration test?

Reply:
I tried to add some of the Fivetran packages to test and it is missing shims for DuckDB. TBH, I feel like testing the overall exclusion strategy is enough in that case.

Reply:
ok!